CHANGELOG.rst

```diff
@@ -5,6 +5,7 @@ Unreleased
 ~~~~~~~~~~
 * Change CDS keys from .cdsapirc file to .config/eracli.txt file. This will avoid conflict with e.g. ADS.
 * Add validator for era5cli.txt keys. This should provide better feedback to users and reduce user error.
+* Automatically split up requests for --land and --ensemble by month. This will prevent users encountering a Request Too Large error.
```

Comment:
It never worked before unless you specified you only want to download a certain number of hours/days.

`era5cli hourly --land --variables soil_temperature_level_1 --startyear 2008` would not work
`era5cli hourly --land --variables soil_temperature_level_1 --startyear 2008 --hours 0` works

I _could_ try to figure out how to determine the maximum number of requests, and how to calculate that, and only (automatically) split the data if that is true. However, that would require a good amount of extra time and depends on the CDS not changing their limits.

It would be difficult and inefficient to try to catch when the cdsapi returns a RequestTooLarge error (or well, a default Exception where we match Request Too Large from the error message). We only find out after making a request, and this is split up over multiple threads.